libs/pages/cluster/src/lib/feature/page-settings-advanced-feature/page-settings-advanced-feature.spec.tsx

```diff
@@ -71,29 +72,29 @@ describe('PageSettingsAdvancedFeature', () => {
   // I think the useForm hook also use useState, this is why the first 4th call are to ignored. https://gist.github.com/mauricedb/eb2bae5592e3ddc64fa965cde4afe7bc
   it('should set the keys if application and advanced_settings are defined', async () => {
     jest.spyOn(React, 'useState').mockImplementation(useStateMock)
-    render(<PageSettingsAdvancedFeature />)
+    renderWithProviders(<PageSettingsAdvancedFeature />)
     expect(setState).toHaveBeenNthCalledWith(9, Object.keys(mockAdvancedSettings || {}).sort())
     await act(async () => {
       await promise
     })
   })
 
   it('should edit ClusterAdvancedSettings if form is submitted', async () => {
-    const { getByLabelText, getByTestId } = render(<PageSettingsAdvancedFeature />)
+    renderWithProviders(<PageSettingsAdvancedFeature />)
 
     await act(() => {
-      fireEvent.input(getByLabelText('loki.log_retention_in_week'), { target: { value: '2' } })
-      fireEvent.input(getByLabelText('aws.vpc.enable_s3_flow_logs'), { target: { value: 'true' } })
-      fireEvent.input(getByLabelText('load_balancer.size'), { target: { value: '/' } })
-      fireEvent.input(getByLabelText('cloud_provider.container_registry.tags'), {
+      fireEvent.input(screen.getByLabelText('loki.log_retention_in_week'), { target: { value: '2' } })
+      fireEvent.input(screen.getByLabelText('aws.vpc.enable_s3_flow_logs'), { target: { value: 'true' } })
+      fireEvent.input(screen.getByLabelText('load_balancer.size'), { target: { value: '/' } })
+      fireEvent.input(screen.getByLabelText('cloud_provider.container_registry.tags'), {
         target: { value: '{"test":"test"}' },
       })
     })
 
-    expect(getByTestId('submit-button')).toBeEnabled()
+    expect(screen.getByTestId('submit-button')).toBeEnabled()
 
     await act(() => {
```

Comment:
Can you remove `act` to be totally iso with our new implementation? or use import not from `'__tests__/utils/setup-jest'` if necessary

Reply:
As discuss, I've moved imports to `@qovery/shared/util-tests` when possible